engine/src/test/java/com/datatorrent/stram/client/AppPackageTest.java

```diff
@@ -105,9 +106,30 @@ public void testAppPackage() throws Exception
 
     Assert.assertEquals("mydtapp-1.0-SNAPSHOT.jar", apps.get("MyFirstApplication").getString("file"));
 
+    String errorStackTrace = apps.get("MyFirstApplication").getString("errorStackTrace");
```

Comment:
It will be OK to remove `apps` map in a separate PR, but it is necessary to avoid any additional dependencies on `apps` that this PR introduces.